templates/default/server.properties.erb

```diff
@@ -136,5 +136,11 @@ zookeeper.connect=<%= node["kafka"]["zookeeper_nodes"].join(?,) %>
 # Timeout in ms for connecting to zookeeper
 zookeeper.connection.timeout.ms=6000
 
+# "Sensible Defaults"
+auto.create.topics.enable=false
+controlled.shutdown.enable=true
+auto.leader.rebalance.enable=true
```

Comment:
`controlled.shutdown.enable=true` and `auto.leader.rebalance.enable=true` are kafka defaults; I could go either way about duplicating them here.